BibBuddy/app/src/main/java/de/bibbuddy/BookNotesView.java

```diff
@@ -170,7 +166,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
   @Override
   public void onPrepareOptionsMenu(Menu menu) {
     MenuItem deleteNote = menu.findItem(R.id.menu_delete_note);
-    deleteNote.setVisible(selectedNoteItems != null && !selectedNoteItems.isEmpty());
+    deleteNote.setVisible(adapter.getSelectedNoteItems().size() > 0);
   }
 
   private void handleDeleteNote() {
```

Comment:
This method does not yield the right singular/plural delete alert - we figured in person that this may be due to the missing referenced counting variable, please fix that :)

Reply:
We talked about the usage of the legacy selectedNoteItems-variable in BookNotesView.
It has been replaced by the adapter-fetching-method and it works that way (means: adapter.getSelectedNoteItems() is correct). I know what you are talking about though and replaced the left over selectedNoteItems-usage the same way.